app/views/govuk_component/docs/search.yml

```diff
@@ -0,0 +1,19 @@
+name: "Search"
+description: "Search box"
+fixtures:
+  default: {}
+  for_use_on_a_dark_but_not_black_background:
+    on_dark_background: true
+  change_label_text:
+    label_text: "Search all the things"
+  stop_the_label_appearing_inline:
+    inline_label: false
+  apply_custom_styling_to_the_label:
+    inline_label: false
+    label_text: "<span class=\"fixture-title\">Search this website</span>"
+  set_id_for_search_input:
+    id: "my_unique_id"
+  set_search_input_value:
+    value: "This is what I searched for"
+  large_version:
```

Comment:
An example with many of the parameters set would be useful, possibly at the start.